src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -288,6 +288,12 @@ public ITaskItem[] ResponseFiles
             get { return (ITaskItem[])_store[nameof(ResponseFiles)]; }
         }
 
+        public string SharedCompilationId
```

Comment:
The only thing that itches me with this PR is whether it needs some documentation. The Roslyn repo has a document for command-line arguments, but none for the Csc/Vbc tasks.

On the other hand, the docs repo has some. 
Here's the page: https://docs.microsoft.com/en-us/visualstudio/msbuild/csc-task
Here's the repo: https://github.com/dotnet/docs/blob/be7974018ce3195dc7344192d647fe64fb2ebcc4/xml/Microsoft.Build.Tasks/Csc.xml

Could you file an issue there to track updating that page once 15.5 is released? You can link it to this PR.